### PR TITLE
Update in onboarding of API

### PR DESCRIPTION
**Pull request by sivakarthikeyanoptit**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ansible/roles/kong-api/defaults/main.yml [hunk 1]
```diff
@@ -6426,3 +6426,39 @@ kong_apis:
...
+    uris: "{{ user_extension_prefix }}/v1/user-extension/programsByPlatformRoles"
+    upstream_url: "{{ ml_core_service_url }}/v1/user-extension/programsByPlatformRoles"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'platformProgramsAccess'
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:6440]: invalid ACL!

File: ansible/roles/kong-api/defaults/main.yml [hunk 2]
```diff
@@ -6426,3 +6426,39 @@ kong_apis:
...
+    uris: "{{ user_extension_prefix }}/v1/user-extension/solutions"
+    upstream_url: "{{ ml_core_service_url }}/v1/user-extension/solutions"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'platformSolutionsAccess'
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:6458]: invalid ACL! Do proivide the correct API Consumer group from existing set as mentioned in same file. Do refer the links for documentation

File: ansible/roles/kong-api/defaults/main.yml (hunk 1 above)
Comment (sivakarthikeyanoptit) [reply ansible/roles/kong-api/defaults/main.yml]: updated

File: ansible/roles/kong-api/defaults/main.yml (hunk 2 above)
Comment (sivakarthikeyanoptit) [reply ansible/roles/kong-api/defaults/main.yml]: update as per suggestion

File: ansible/roles/kong-api/defaults/main.yml (hunk 1 above)
Comment (SMYALTAMASH) [reply ansible/roles/kong-api/defaults/main.yml]: programAccess should be the ACL

File: ansible/roles/kong-api/defaults/main.yml [hunk 3]
```diff
@@ -6426,3 +6426,39 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ small_request_size_limit }}" 
+
+  - name: platformPrograms
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:6430]: give a meaningful api name

**Review by SMYALTAMASH — APPROVED** [review]

**Pull request merged**